src/content/articles/index.js

```diff
@@ -0,0 +1,22 @@
+// ARTICLE IMPORTS
+// local articles
+import testArticle from './local/test-id';
+
+/** 
+ * An indexing of id to relative path of where to access article
+ * After importing the article above, place it in here to map the id
+ */
+export const articleIndex = {
+    [testArticle.id]: testArticle
+};
+
+/**
+ * Basically a mock backend service to retrieve an article given an id
+ * NOTE: currently relies on article actually existing
+ * @param {string} articleId 
+ * @returns article
+ */
+export const getArticleById = async (articleId) => {
+    const article = articleIndex[articleId];
+    return article;
+}
```

Comment:
this is going to quickly become a very messy file with the more articles that we add but basically my idea was to just import articles and then stick them in `articleIndex` to lookup articles by their id then allow `getArticleById` to serve as a mock backend call to retrieve the articles

relying on `getArticleById` rather than the `articleIndex` allows us to eventually change this implementation if we want to, as long as getArticleById guarantees that it will somehow return an article when we pass it an id, whether we actually spin up a mock backend or keep relying on these locally stored articles and imports